doc/api/n-api.md

```diff
@@ -3313,6 +3313,8 @@ support it:
 * If the function is not available, provide an alternate implementation
   that does not use the function.
 
+<!-- it's very convenient to have all the anchors indexed -->
```

Comment:
The comment? I felt like it wasn't obvious why the added `disable no-unused-definitions`, I can rephrase, also happy to take suggestions.